packages/tao/src/shared/tree.ts

```diff
@@ -272,6 +280,35 @@ export class FsTree implements Tree {
     return res;
   }
 
+  updateFilePermissions(filePath: string, mode: string | number): void {
```

Comment:
Let's call it `changePermissions` instead.

It's closer to `chmod`. I don't think we need `File` in the name since this is a representation of a file tree already. We have `read` instead of `readFile` and `write` instead of `writeFile`.

Reply:
Makes sense 👍🏻